.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+# Description
+Briefly describe the problem or reason for the changes implemented in the PR
+
+# Trac ticket
+Link to the Trac ticket for this PR (all PRs except really trivial issues or 
+tests adding need an accepted ticket).
+
+# UI Changes
+Please attach screenshots in light and dark mode for UI changes.
+
+# Checklist
+- [ ] I have updated the `has patch` flag in the Trac ticket
+- [ ] The commit message is written in past tense, mentions the ticket #, and
+ ends with a dot
```

Comment:
It’s [“period” in American English](https://en.wikipedia.org/wiki/Full_stop), “dot” would only refer to use as an operator in code, like `self.cheese`. Also add one here.
```suggestion
 ends with a period.
```